app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transferdetails/components/ExistingTransferFilesDetailsScreen.kt

```diff
@@ -25,6 +25,7 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.platform.LocalContext
 import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.infomaniak.multiplatform_swisstransfer.common.models.TransferDirection
```

Comment:
Clean the project.

<img width="564" height="545" alt="Image" src="https://github.com/user-attachments/assets/bc66cb5d-2fbd-4564-96a2-1af60927a7d3" />